src/mono/mono/mini/llvm-runtime.cpp

```diff
@@ -10,6 +15,9 @@ mono_llvm_cpp_throw_exception (void)
 {
 	gint32 *ex = NULL;
 
+	if (mono_trace_is_enabled ())
+		mono_runtime_printf_err ("Native Stacktrace (mono_llvm_cpp_throw_exception)"); 
```

Comment:
Emscripten needs a newline to flush to the debugger console.

```suggestion
		mono_runtime_printf_err ("Native Stacktrace (mono_llvm_cpp_throw_exception)\n"); 
```

Reply:
And it's likely to be the same here:

https://github.com/dotnet/runtime/blob/c7cfa33ffe19d80b709df4565365b51cd64aab21/src/mono/mono/mini/mini-exceptions.c#L3439